<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>authorization</class>
 <widget class="QWidget" name="authorization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(
    spread:pad, 
    x1:0, y1:0, x2:1, y2:0, 
    stop:0 rgba(10, 10, 140, 70), 
    stop:1 rgba(10, 10, 140, 70)
);</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>601</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="loginButton">
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);
background: rgb(20, 30, 48)</string>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="regButton">
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);
background: rgb(20, 30, 48)</string>
        </property>
        <property name="text">
         <string>Registration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="changeButton">
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);
background: rgb(20, 30, 48)</string>
        </property>
        <property name="text">
         <string>Change</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_login">
      <property name="styleSheet">
       <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Login:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="styleSheet">
       <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEdit_login">
      <property name="styleSheet">
       <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_password">
      <property name="styleSheet">
       <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
